src/components/UploadIdsModal.js

```diff
@@ -61,7 +61,7 @@ const UploadIdsModal = ({
             setSQON(nextSQON);
             unsetModal();
           },
-          submitText: 'Upload',
```

Comment:
@rosibaj was this part of the ticket?

Reply:
thanks @hlminh2000 looks fine lets merge